app/src/pages/instance/pluginsPage/availablePluginDetail/availablePluginDetail.scss

```
@@ -0,0 +1,183 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.available-plugin-detail {
+  display: flex;
+  padding-left: 5px;
+  // ui-kit: OpenSans (body default --rp-ui-base-font-family is Roboto)
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)